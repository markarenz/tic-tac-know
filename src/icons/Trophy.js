import React from 'react';

const Trophy = () => {
  return (
    <svg xmlns="http://www.w3.org/2000/svg" version="1.1" viewBox="0 0 76 100" xmlSpace="preserve">
      <path
        fill="#f5ac51"
        strokeWidth="0.191"
        d="M75.122 1.083a1.585 1.585 0 00-1.21-.56h-5.285a1.585 1.585 0 100 3.17h3.413l-3.63 21.782c-1.124 6.744-7.6 12.23-14.436 12.23h-16v3.17h16c8.317 0 16.196-6.674 17.564-14.878l3.938-23.628c.077-.46-.053-.93-.354-1.286z"
      ></path>
      <path
        fill="#ffd159"
        strokeWidth="0.191"
        d="M21.974 37.705c-6.837 0-13.313-5.486-14.437-12.23L3.907 3.694h3.414a1.585 1.585 0 100-3.171H2.036A1.585 1.585 0 00.472 2.369L4.41 25.997c1.367 8.204 9.246 14.879 17.564 14.879h16v-3.171z"
      ></path>
      <path
        fill="#f5ac51"
        d="M239.387 366.04L272.619 366.04 272.619 307.395 256.003 296.318 239.387 307.395z"
        transform="translate(-10.882 .523) scale(.19084)"
      ></path>
      <path
        fill="#c9740c"
        fillOpacity="1"
        strokeWidth="0.191"
        d="M34.802 48.181H41.144V59.186H34.802z"
      ></path>
      <path
        fill="#f5ac51"
        strokeWidth="0.191"
        d="M53.829 2.108L37.974 52.844c7.557 0 14.757-6.099 16-13.554L60.17 2.108z"
      ></path>
      <path
        fill="#ffd159"
        strokeWidth="0.191"
        d="M49.402 39.29L53.83 2.108H15.777l6.197 37.182c1.242 7.455 8.442 13.554 16 13.554 5.398 0 10.54-6.099 11.428-13.554z"
      ></path>
      <path
        fill="#f5ac51"
        strokeWidth="0.191"
        d="M37.974 65.466l3.17 13.741 6.343-4.228a9.513 9.513 0 00-9.513-9.513z"
      ></path>
      <path
        fill="#ffd159"
        strokeWidth="0.191"
        d="M37.974 65.466a9.513 9.513 0 00-9.513 9.513l12.684 4.228V74.98c0-5.254-1.42-9.513-3.171-9.513z"
      ></path>
      <path
        fill="#f1866d"
        strokeWidth="0.191"
        d="M47.487 74.98h-6.342L49.6 87.662h6.342v-4.228a8.456 8.456 0 00-8.456-8.456z"
      ></path>
      <path
        fill="#f5ac51"
        strokeWidth="0.191"
        d="M41.145 74.98H28.46a8.456 8.456 0 00-8.456 8.455v4.228H49.6v-4.228a8.456 8.456 0 00-8.456-8.456z"
      ></path>
      <path
        fill="#8a4f08"
        fillOpacity="1"
        d="M328.005 473.229L350.16 512 383.392 512 383.392 434.459 350.16 434.459z"
        transform="translate(-10.882 .523) scale(.19084)"
      ></path>
      <path
        fill="#c9740c"
        fillOpacity="1"
        strokeWidth="0.191"
        d="M13.662 83.436H55.942V98.23400000000001H13.662z"
      ></path>
      <path
        fill="#ffd159"
        strokeWidth="0.191"
        d="M41.673 60.772h-7.399a1.585 1.585 0 110-3.171h7.4a1.585 1.585 0 110 3.17z"
      ></path>
      <path
        d="M32.541 21.031h-2.176l-.477 2.198h1.953v1.605H29.53l-.686 3h-1.61l.7-3h-1.788l-.67 3h-1.61l.685-3h-1.67v-1.605h2.028l.492-2.198h-1.82v-1.508h2.162l.701-3.08h1.61l-.715 3.08h1.788l.686-3.08h1.61l-.7 3.08h1.818zm-4.248 2.198l.477-2.198h-1.774l-.492 2.198z"
        style={{}}
        fill="#c9740c"
        fillOpacity="1"
        stroke="#000"
        strokeOpacity="0"
        strokeWidth="0.161"
        fontFamily="Raleway"
        fontSize="192"
        fontWeight="bold"
      ></path>
      <path
        d="M47.969 29.746v4.722H34.627v-4.722h4.667V17.923q-.296.468-.92.976-.625.468-1.414.897-.788.43-1.577.703-.789.273-1.413.273v-4.878q.592 0 1.413-.429.854-.468 1.676-1.093.854-.624 1.446-1.21.624-.624.789-.975h4.501v17.56z"
        style={{}}
        fill="#c9740c"
        fillOpacity="1"
        stroke="#000"
        strokeOpacity="0"
        strokeWidth="0.373"
        fontFamily="Raleway"
        fontSize="192"
        fontWeight="bold"
      ></path>
      <path
        fill="#f5ac51"
        strokeWidth="0.191"
        d="M60.7 3.694H15.247a1.585 1.585 0 110-3.171H60.7a1.585 1.585 0 110 3.17z"
      ></path>
    </svg>
  );
};

export default Trophy;
