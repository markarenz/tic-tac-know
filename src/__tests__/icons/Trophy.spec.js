import { render, screen } from '@testing-library/react';
import Trophy from '../../icons/Trophy';

describe('Trophy', () => {
  it('renders component', () => {
    const container = render(<Trophy />);
    expect(container).toMatchSnapshot();
  });
});
