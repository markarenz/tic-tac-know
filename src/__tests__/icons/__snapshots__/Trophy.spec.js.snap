// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Trophy renders component 1`] = `
Object {
  "asFragment": [Function],
  "baseElement": <body>
    <div>
      <svg
        version="1.1"
        viewBox="0 0 76 100"
        xml:space="preserve"
        xmlns="http://www.w3.org/2000/svg"
      >
        <path
          d="M75.122 1.083a1.585 1.585 0 00-1.21-.56h-5.285a1.585 1.585 0 100 3.17h3.413l-3.63 21.782c-1.124 6.744-7.6 12.23-14.436 12.23h-16v3.17h16c8.317 0 16.196-6.674 17.564-14.878l3.938-23.628c.077-.46-.053-.93-.354-1.286z"
          fill="#f5ac51"
          stroke-width="0.191"
        />
        <path
          d="M21.974 37.705c-6.837 0-13.313-5.486-14.437-12.23L3.907 3.694h3.414a1.585 1.585 0 100-3.171H2.036A1.585 1.585 0 00.472 2.369L4.41 25.997c1.367 8.204 9.246 14.879 17.564 14.879h16v-3.171z"
          fill="#ffd159"
          stroke-width="0.191"
        />
        <path
          d="M239.387 366.04L272.619 366.04 272.619 307.395 256.003 296.318 239.387 307.395z"
          fill="#f5ac51"
          transform="translate(-10.882 .523) scale(.19084)"
        />
        <path
          d="M34.802 48.181H41.144V59.186H34.802z"
          fill="#c9740c"
          fill-opacity="1"
          stroke-width="0.191"
        />
        <path
          d="M53.829 2.108L37.974 52.844c7.557 0 14.757-6.099 16-13.554L60.17 2.108z"
          fill="#f5ac51"
          stroke-width="0.191"
        />
        <path
          d="M49.402 39.29L53.83 2.108H15.777l6.197 37.182c1.242 7.455 8.442 13.554 16 13.554 5.398 0 10.54-6.099 11.428-13.554z"
          fill="#ffd159"
          stroke-width="0.191"
        />
        <path
          d="M37.974 65.466l3.17 13.741 6.343-4.228a9.513 9.513 0 00-9.513-9.513z"
          fill="#f5ac51"
          stroke-width="0.191"
        />
        <path
          d="M37.974 65.466a9.513 9.513 0 00-9.513 9.513l12.684 4.228V74.98c0-5.254-1.42-9.513-3.171-9.513z"
          fill="#ffd159"
          stroke-width="0.191"
        />
        <path
          d="M47.487 74.98h-6.342L49.6 87.662h6.342v-4.228a8.456 8.456 0 00-8.456-8.456z"
          fill="#f1866d"
          stroke-width="0.191"
        />
        <path
          d="M41.145 74.98H28.46a8.456 8.456 0 00-8.456 8.455v4.228H49.6v-4.228a8.456 8.456 0 00-8.456-8.456z"
          fill="#f5ac51"
          stroke-width="0.191"
        />
        <path
          d="M328.005 473.229L350.16 512 383.392 512 383.392 434.459 350.16 434.459z"
          fill="#8a4f08"
          fill-opacity="1"
          transform="translate(-10.882 .523) scale(.19084)"
        />
        <path
          d="M13.662 83.436H55.942V98.23400000000001H13.662z"
          fill="#c9740c"
          fill-opacity="1"
          stroke-width="0.191"
        />
        <path
          d="M41.673 60.772h-7.399a1.585 1.585 0 110-3.171h7.4a1.585 1.585 0 110 3.17z"
          fill="#ffd159"
          stroke-width="0.191"
        />
        <path
          d="M32.541 21.031h-2.176l-.477 2.198h1.953v1.605H29.53l-.686 3h-1.61l.7-3h-1.788l-.67 3h-1.61l.685-3h-1.67v-1.605h2.028l.492-2.198h-1.82v-1.508h2.162l.701-3.08h1.61l-.715 3.08h1.788l.686-3.08h1.61l-.7 3.08h1.818zm-4.248 2.198l.477-2.198h-1.774l-.492 2.198z"
          fill="#c9740c"
          fill-opacity="1"
          font-family="Raleway"
          font-size="192"
          font-weight="bold"
          stroke="#000"
          stroke-opacity="0"
          stroke-width="0.161"
        />
        <path
          d="M47.969 29.746v4.722H34.627v-4.722h4.667V17.923q-.296.468-.92.976-.625.468-1.414.897-.788.43-1.577.703-.789.273-1.413.273v-4.878q.592 0 1.413-.429.854-.468 1.676-1.093.854-.624 1.446-1.21.624-.624.789-.975h4.501v17.56z"
          fill="#c9740c"
          fill-opacity="1"
          font-family="Raleway"
          font-size="192"
          font-weight="bold"
          stroke="#000"
          stroke-opacity="0"
          stroke-width="0.373"
        />
        <path
          d="M60.7 3.694H15.247a1.585 1.585 0 110-3.171H60.7a1.585 1.585 0 110 3.17z"
          fill="#f5ac51"
          stroke-width="0.191"
        />
      </svg>
    </div>
  </body>,
  "container": <div>
    <svg
      version="1.1"
      viewBox="0 0 76 100"
      xml:space="preserve"
      xmlns="http://www.w3.org/2000/svg"
    >
      <path
        d="M75.122 1.083a1.585 1.585 0 00-1.21-.56h-5.285a1.585 1.585 0 100 3.17h3.413l-3.63 21.782c-1.124 6.744-7.6 12.23-14.436 12.23h-16v3.17h16c8.317 0 16.196-6.674 17.564-14.878l3.938-23.628c.077-.46-.053-.93-.354-1.286z"
        fill="#f5ac51"
        stroke-width="0.191"
      />
      <path
        d="M21.974 37.705c-6.837 0-13.313-5.486-14.437-12.23L3.907 3.694h3.414a1.585 1.585 0 100-3.171H2.036A1.585 1.585 0 00.472 2.369L4.41 25.997c1.367 8.204 9.246 14.879 17.564 14.879h16v-3.171z"
        fill="#ffd159"
        stroke-width="0.191"
      />
      <path
        d="M239.387 366.04L272.619 366.04 272.619 307.395 256.003 296.318 239.387 307.395z"
        fill="#f5ac51"
        transform="translate(-10.882 .523) scale(.19084)"
      />
      <path
        d="M34.802 48.181H41.144V59.186H34.802z"
        fill="#c9740c"
        fill-opacity="1"
        stroke-width="0.191"
      />
      <path
        d="M53.829 2.108L37.974 52.844c7.557 0 14.757-6.099 16-13.554L60.17 2.108z"
        fill="#f5ac51"
        stroke-width="0.191"
      />
      <path
        d="M49.402 39.29L53.83 2.108H15.777l6.197 37.182c1.242 7.455 8.442 13.554 16 13.554 5.398 0 10.54-6.099 11.428-13.554z"
        fill="#ffd159"
        stroke-width="0.191"
      />
      <path
        d="M37.974 65.466l3.17 13.741 6.343-4.228a9.513 9.513 0 00-9.513-9.513z"
        fill="#f5ac51"
        stroke-width="0.191"
      />
      <path
        d="M37.974 65.466a9.513 9.513 0 00-9.513 9.513l12.684 4.228V74.98c0-5.254-1.42-9.513-3.171-9.513z"
        fill="#ffd159"
        stroke-width="0.191"
      />
      <path
        d="M47.487 74.98h-6.342L49.6 87.662h6.342v-4.228a8.456 8.456 0 00-8.456-8.456z"
        fill="#f1866d"
        stroke-width="0.191"
      />
      <path
        d="M41.145 74.98H28.46a8.456 8.456 0 00-8.456 8.455v4.228H49.6v-4.228a8.456 8.456 0 00-8.456-8.456z"
        fill="#f5ac51"
        stroke-width="0.191"
      />
      <path
        d="M328.005 473.229L350.16 512 383.392 512 383.392 434.459 350.16 434.459z"
        fill="#8a4f08"
        fill-opacity="1"
        transform="translate(-10.882 .523) scale(.19084)"
      />
      <path
        d="M13.662 83.436H55.942V98.23400000000001H13.662z"
        fill="#c9740c"
        fill-opacity="1"
        stroke-width="0.191"
      />
      <path
        d="M41.673 60.772h-7.399a1.585 1.585 0 110-3.171h7.4a1.585 1.585 0 110 3.17z"
        fill="#ffd159"
        stroke-width="0.191"
      />
      <path
        d="M32.541 21.031h-2.176l-.477 2.198h1.953v1.605H29.53l-.686 3h-1.61l.7-3h-1.788l-.67 3h-1.61l.685-3h-1.67v-1.605h2.028l.492-2.198h-1.82v-1.508h2.162l.701-3.08h1.61l-.715 3.08h1.788l.686-3.08h1.61l-.7 3.08h1.818zm-4.248 2.198l.477-2.198h-1.774l-.492 2.198z"
        fill="#c9740c"
        fill-opacity="1"
        font-family="Raleway"
        font-size="192"
        font-weight="bold"
        stroke="#000"
        stroke-opacity="0"
        stroke-width="0.161"
      />
      <path
        d="M47.969 29.746v4.722H34.627v-4.722h4.667V17.923q-.296.468-.92.976-.625.468-1.414.897-.788.43-1.577.703-.789.273-1.413.273v-4.878q.592 0 1.413-.429.854-.468 1.676-1.093.854-.624 1.446-1.21.624-.624.789-.975h4.501v17.56z"
        fill="#c9740c"
        fill-opacity="1"
        font-family="Raleway"
        font-size="192"
        font-weight="bold"
        stroke="#000"
        stroke-opacity="0"
        stroke-width="0.373"
      />
      <path
        d="M60.7 3.694H15.247a1.585 1.585 0 110-3.171H60.7a1.585 1.585 0 110 3.17z"
        fill="#f5ac51"
        stroke-width="0.191"
      />
    </svg>
  </div>,
  "debug": [Function],
  "findAllByAltText": [Function],
  "findAllByDisplayValue": [Function],
  "findAllByLabelText": [Function],
  "findAllByPlaceholderText": [Function],
  "findAllByRole": [Function],
  "findAllByTestId": [Function],
  "findAllByText": [Function],
  "findAllByTitle": [Function],
  "findByAltText": [Function],
  "findByDisplayValue": [Function],
  "findByLabelText": [Function],
  "findByPlaceholderText": [Function],
  "findByRole": [Function],
  "findByTestId": [Function],
  "findByText": [Function],
  "findByTitle": [Function],
  "getAllByAltText": [Function],
  "getAllByDisplayValue": [Function],
  "getAllByLabelText": [Function],
  "getAllByPlaceholderText": [Function],
  "getAllByRole": [Function],
  "getAllByTestId": [Function],
  "getAllByText": [Function],
  "getAllByTitle": [Function],
  "getByAltText": [Function],
  "getByDisplayValue": [Function],
  "getByLabelText": [Function],
  "getByPlaceholderText": [Function],
  "getByRole": [Function],
  "getByTestId": [Function],
  "getByText": [Function],
  "getByTitle": [Function],
  "queryAllByAltText": [Function],
  "queryAllByDisplayValue": [Function],
  "queryAllByLabelText": [Function],
  "queryAllByPlaceholderText": [Function],
  "queryAllByRole": [Function],
  "queryAllByTestId": [Function],
  "queryAllByText": [Function],
  "queryAllByTitle": [Function],
  "queryByAltText": [Function],
  "queryByDisplayValue": [Function],
  "queryByLabelText": [Function],
  "queryByPlaceholderText": [Function],
  "queryByRole": [Function],
  "queryByTestId": [Function],
  "queryByText": [Function],
  "queryByTitle": [Function],
  "rerender": [Function],
  "unmount": [Function],
}
`;
